### ISSUE TITLE
[Bug]: Failed to sync but I have no idea why

### ISSUE THREAD

**Issue by gazm2k5** [remotely-save/remotely-save#883]
### What happened?

I just installed Obsidian and am using Nextcloud so figured I'd use remotelysave to sync.

I installed on my Windows PC and Android phone and everything seemed to be working and syncing.

Earlier today I used my phone to make some changes, including deleting a bunch of notes I made that were basically just "Test". This evening I came home and I can see that my Nextcloud instance has correctly synced all the changes I made on mobile, however on my PC I can still see my old "Test" notes that I deleted. In the corner it just says "failed to sync" and doesn't tell me why.

I checked the settings and tested the connection and it said it could successfully contact my webdav server.

How do I even begin troubleshooting this given there are no errors messages displayed?

### What OS are you using?

Windows, Android

### What remote cloud services are you using? (Please choose the specified one if it's in the list)

webdav (NextCloud)

### Version of the plugin

0.5.25

### Version of Obsidian

1.7.5

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by fyears** [comment]
please trigger a manual sync and see any error messages

**Comment by gazm2k5** [comment]
> please trigger a manual sync and see any error messages

I don't see a manual sync button but I have it set to sync every minute which it tries to do but then says failed. No error messages anywhere.

**Comment by gazm2k5** [comment]
I disabled the plugin on PC, removed all my local files, then renabled the plugin. It correctly synced and downloaded files from Nextcloud.

I then created a new file on mobile and now the same "failed to sync" error is happening on my PC. It seems it doesn't like any changes to be made. I do have bidirectional sync turned on though.

**Comment by rekrapnella** [comment]
tl;dr I had a similar issue, but it was because of the "Abort Sync If Modification Above Percentage" setting. Changing the setting (from  '50 (default)' to '100 (disable the protection) seems to have resolved the issue without breaking anything.

I don't know what your exact settings look like, but I ran into the same issue. In working to repair it I opened the debug [ctrl+shift+i] and switching to the console, I found an error from the plugin that aborted the sync because of the "Abort Sync If Modification Above Percentage" setting. Changing the setting resolved the issue and allowed the sync to go through. The pages on my mobile device were not so altered from the overall changes that I can see now that the sync has gone through, and I can find no explicit thing that has changed greatly. I'm not highly skilled in these issues, but my best guesses are that it's either, 1. the differences for the mobile and the desktop app are interpreted by Remotely Save as significant, even though they're not apparently different on the face of things, or 2. some difference in plugins or visual formatting are causing the issue, but are then not material when recreating the pages.

**Comment by Damaskox** [comment]
The debug window really did help out solving my own sync error problem (as what rekrapnella noted).
In my case it was an error with some attachment files (that I deleted).
